.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,6 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
❓ I am still unsure why this file changes sometimes and sometimes doesn't. Should we exclude the changes from this PR?

Reply:
Maybe `Project.xml` is the default for project code style? Let's remove the line again from this PR and only change the other file.

Reply:
Alright, fine with me as well. FWIW, the GE project has `<option name="PREFERRED_PROJECT_CODE_STYLE" value="Project" />` in there. Is that maybe what we should do as well?

Reply:
I don't feel strongly. This seems to work well. If we bump into more problems, let's re-assess.